master/Makefile

```diff
@@ -63,7 +63,8 @@ build/mock_gen.stamp: $(MOCK_GENERATION_INPUTS)
 	mockery --name=ExperimentAuthZ --dir=internal/experiment --output internal/mocks --filename authz_experiment_iface.go
 	mockery --name=NSCAuthZ --dir=internal/command --output internal/mocks --filename nsc_authz_iface.go
 	mockery --name=ModelAuthZ --dir=internal/model --output internal/mocks --filename authz_model_iface.go
-
+	mockery --name=PodInterface --dir=${HOME}/go/pkg/mod/k8s.io/client-go@v0.20.14/kubernetes/typed/core/v1 --name="PodInterface" --filename=pod_iface.go --output=master/internal/mocks --outpkg=mocks          
+	mockery --name=EventInterface --dir=${HOME}/go/pkg/mod/k8s.io/client-go@v0.20.14/kubernetes/typed/core/v1 --name="EventInterface" --filename=event_iface.go --output=master/internal/mocks --outpkg=mocks 
```

Comment:
Currently running `make clean && make all` deletes all generated mock files -- I think it's because I'm not correctly generating the Pod/EventInterface mocks in this Makefile. Need some help!

Reply:
`mockery --name=PodInterface --srcpkg='k8s.io/client-go/kubernetes/typed/core/v1'`
should work. but then we need a mock gen input that takes into account the package version — for now i would just make go.sum and go.mod mock gen inputs